crates/subspace-archiving/src/archiver.rs

```diff
@@ -179,6 +179,10 @@ pub enum ArchiverInstantiationError {
 /// records of `RECORD_SIZE`, records are erasure coded, Merkle Tree is built over them, and
 /// with Merkle Proofs appended records become pieces that are returned alongside corresponding root
 /// block header.
+///
+/// ## Panics
+/// Panics when operating on blocks, whose length doesn't fit into u32 (should never be the case in
```

Comment:
If I understand correctly, `length` means the block number, `length` is not super explicit IMHO.
```suggestion
/// Panics when operating on blocks, whose block number doesn't fit into u32 (should never be the case in
```

Reply:
It is about the length of encoded data, not block number